### ISSUE TITLE
How to add data augmentation in clients' local datasets?

### ISSUE THREAD

**Issue by dmndxld** [adap/flower#3070]
### What is your question?

Hello! I want to add data augmentation in clients' local datasets after spilt different clients. For example, add rotation or translation in clients' mnist datasets. Im new to flower framework. Now im reading 
`baselines/flwr_baselines/flwr_baselines/publications/fedavg_mnist `.
In this folder, how to add this operation? MANY THANKS IN ADVANCE!!!

**Comment by xiaoToby** [comment]
good question, I also want to get good answers

**Comment by WilliamLindskog** [comment]
Yes, you can use `flwr-datasets` to add data augmentation. After partitioning the MNIST dataset using `flwr-datasets.partition`, apply data augmentation to each client’s partition with `flwr_datasets.with_transform` and torchvision.transforms. You can find an example [here](https://github.com/adap/flower/blob/main/examples/advanced-pytorch/pytorch_example/task.py) on how to use `flwr-datasets` and transforms. 

`flwr-datasets` simplifies partitioning, augmentation, and scaling to multiple clients for federated learning. 

Closing this issue for now.
